java/style-wms-layer/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <ToggleButton
+        android:id="@+id/toggleStyleButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="24dp"
```

Comment:
The button covers the "Licensed For Developer Use Only", I do `android:layout_marginBottom="45dp"` to set it on top

Reply:
This is a common problem. The "Licensed for Developer Use Only" thing moves around everytime the app is run. So there's sort of no getting around it.